bolt/lib/Utils/CommandLineOpts.cpp

```diff
@@ -11,7 +11,7 @@
 //===----------------------------------------------------------------------===//
 
 #include "bolt/Utils/CommandLineOpts.h"
-#include "llvm/Support/VCSRevision.h"
+#include "VCSVersion.inc"
 
 using namespace llvm;
 
```

Comment:
below in "#ifdef LLVM_REVISION    LLVM_REVISION"
you need to change that to BOLT_REVISION (both instances)

Reply:
Done